common/culture/cultures/orc.txt

```diff
@@ -54,9 +54,9 @@
 		mother_name_chance = 5
```

Comment:
`durotaric` culture still has `green_orcish` as ethnicity.

Reply:
Fixed in 5d46d4eb7a4803e49bf117c178878798e9bbc319.